android/src/toga_android/widgets/activityindicator.py

```diff
@@ -0,0 +1,43 @@
+from android import R
+from android.view import View
+from android.widget import ProgressBar
+
+from .base import Widget
+
+
+class ActivityIndicator(Widget):
+    def create(self):
+        self.native = ProgressBar(
+            self._native_activity,
+            None,
+            R.attr.progressBarStyleSmall,
+        )
+        # Always set as running and we just control visibility.
+        self.native.setIndeterminate(True)
+        self._running = False
+
+    def is_running(self):
+        return self._running
+
+    def start(self):
+        self._running = True
+        self.native.setVisibility(View.INVISIBLE if self._hidden else View.VISIBLE)
```

Comment:
This potentially uses `self._hidden` before it is defined. It evidently works, but it's risky because it's sensitive to a specific implementation.